libs/ledger-live-common/src/families/ethereum/signOperation.ts

```diff
@@ -46,16 +53,28 @@ export const signOperation = ({
             async function main() {
               // First, we need to create a partial tx and send to the device
               const { freshAddressPath, freshAddress } = account;
-              const { gasPrice } = transaction;
+              const { gasPrice, maxBaseFeePerGas, maxPriorityFeePerGas } =
+                transaction;
               const gasLimit = getGasLimit(transaction);
 
-              if (!gasPrice || !new BigNumber(gasLimit).gt(0)) {
+              let dataMissing = !new BigNumber(gasLimit).gt(0);
+              let missingDataString = "";
+              if (EIP1559ShouldBeUsed(account.currency)) {
+                missingDataString = ` maxBaseFeePerGas=${String(
```

Comment:
FYI: if `maxBaseFeePerGas` is `null` or `undefined` then `String(maxBaseFeePerGas)` will output `"null"` or `"undefined"`. If that's not what we want it might be safer to check if value is defined and use `.toString`